marconi-sidechain/src/Marconi/Sidechain/Api/Query/Indexers/EpochSPD.hs

```diff
@@ -0,0 +1,52 @@
+module Marconi.Sidechain.Api.Query.Indexers.EpochSPD
+    ( initializeEnv
+    , updateEnvState
+    , querySPDByEpochNo
+    ) where
+
+import Cardano.Api qualified as C
+import Control.Concurrent.STM.TMVar (TMVar, newEmptyTMVarIO, tryReadTMVar)
+import Control.Lens ((^.))
+import Control.Monad.STM (STM, atomically)
+import Data.Word (Word64)
+import Marconi.ChainIndex.Indexers.EpochStakepoolSize (EpochSPDHandle, StorableQuery (SPDByEpochNoQuery),
+                                                       StorableResult (SPDByEpochNoResult))
+import Marconi.Core.Storable (State)
+import Marconi.Core.Storable qualified as Storable
+import Marconi.Sidechain.Api.Routes (EpochStakePoolDelegationResult (EpochStakePoolDelegationResult))
+import Marconi.Sidechain.Api.Types (EpochSPDIndexerEnv (EpochSPDIndexerEnv), QueryExceptions (QueryError), SidechainEnv,
+                                    epochSpdIndexerEnvIndexer, sidechainEnvIndexers,
+                                    sidechainEpochStakePoolDelegationIndexer)
+import Marconi.Sidechain.Utils (writeTMVar)
+
+-- | Bootstraps the EpochSPD query environment.
```

Comment:
this doesn't apply any more since we are on Sqlite WALL.  We should replace, or take it out.
I'll update the utxo  part  on my PR as well since it is also there.

Reply:
Sorry I don't understand the comment. What do that have to do with `initializeEnv`?